### ISSUE TITLE
forums and subforums

### ISSUE THREAD

**Issue by inrussiawithhate** [brunnopleffken/addictive-community#36]
Hello! Can you add and parenting forums, for example the picture? Not only rooms, but also split them.

![asd](https://cloud.githubusercontent.com/assets/7196689/8325040/d540c52e-1a5e-11e5-8dfd-690788b9174f.jpg)

**Comment by halojoy** [comment]
One way would be to have Categories.
The possibility to group several rooms into one category.

**Comment by brunnopleffken** [comment]
Sounds nice, and not hard do implement at all! ;)
If everything works fine, I think this may come in the next release...

**Comment by brunnopleffken** [comment]
Yeah! This is how it looks!

![screen shot 2015-06-30 at 12 52 33 am](https://cloud.githubusercontent.com/assets/3427344/8423617/4f56b50a-1ec2-11e5-8f09-cf8b8f890f6f.png)

**Comment by halojoy** [comment]
Good work, Brunno.
When I wrote my forum script, I have an admin function for DisplayOrder. Just by changing a number I can move a forum up and down in the frontpage.

**Comment by brunnopleffken** [comment]
Actually there is a field for displaying order that hasn't been yet programmed...

```sql
CREATE TABLE `c_categories` (
  `c_id` int(2) NOT NULL AUTO_INCREMENT,
  `name` varchar(30) NOT NULL DEFAULT '',
  `order_n` int(2) DEFAULT NULL,
  `visible` int(1) NOT NULL,
  PRIMARY KEY (`c_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

I want to create something like this: https://jqueryui.com/sortable/
Drag and drop will make thing a lot easier!

Another enhancement that I want to do is to not show any category when there's only one category or the others are empty. In this case, there isn't any division, the "category" is the community itself.

**Comment by halojoy** [comment]
That is most convinient. Drag and drop.
In many places I have seen the use of up-arrow and down-arrows.
I wanted it simple, so I made to fill in numbers in a form input field: 2,1,4,3 etc.

**Comment by brunnopleffken** [comment]
I was thinking... For now I think I'll proceed this way: numbers in an input field.
So I can release v0.6.0 as soon as possible with all those fixes we did last week.

I'll create this drag-and-drop stuff as an enhancement (v 0.6.1 or 0.7.0).

**Comment by halojoy** [comment]
I did a foreach inside the form. Which produce an array in $_POST.
This input array is used to UPDATE the ranking field of forums.
Yes.
You can always fix a drag and drop later. This is something I could not do :)
I can only use PHP in my scripts.
I don't think I will have to show my code. I am sure you know how.

**Comment by brunnopleffken** [comment]
That's it! After a few days busy with other hard works, the commit 087444c98e61b1329bd376914cb524d49f63c5fa closes this improvement suggestion. This will be released soon in v0.6.0.

:)
